### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by slafayIGN**
## Description

Add translation in french

## Related issues or pull requests

None

## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [BSD 2-Clause License](https://github.com/geostyler/geostyler/blob/master/)
- [x] I have followed the [guidelines for contributing](https://github.com/geostyler/geostyler/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/geostyler/geostyler/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

File: src/locale/fr_FR.ts [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+    add: 'Ajouter',
+    remove: 'Supprimer'
+  },
+  GsUploadButton: {
+    upload: 'Téléverser'
+  },
+  GsFilterTree: {
+    andDrpdwnLabel: 'AND-Filter',
+    orDrpdwnLabel: 'OR-Filter',
+    notDrpdwnLabel: 'NOT-Filter',
```
Comment (jansule) [inline src/locale/fr_FR.ts:273]: Is it intentional to use the `-Filter` suffix here instead of something like `-filtre`?

**Review by jansule — APPROVED** [review]
Very nice work @slafayIGN! I have just a minor question

File: src/locale/fr_FR.ts (hunk 1 above)
Comment (slafayIGN) [reply src/locale/fr_FR.ts:273]: I was unsure of how I would like it to appear in the UI
So there are still some strings in english, but not much.

Comment (jansule) [reply src/locale/fr_FR.ts:273]: Alright, no problem. This is definitely unblocking for me

**Pull request merged**